docs/devel/cherry-picks.md

```diff
@@ -49,6 +49,7 @@ depending on the point in the release cycle.
   * Add an appropriate release-note-(!label-needed) label
   * Add an appropriate milestone
   * Add the `cherrypick-candidate` label
+  * The PR title is the **release note** you want published at release time
```

Comment:
@roberthbailey had a question about when the PR title is grabbed for release notes. For instance, is it possible to change the PR title of the master PR when assembling the release notes if the person building the release does a dry run and doesn't think the release notes are adequate?